doc/whats_new/v1.1.rst

```diff
@@ -96,6 +96,13 @@ Changelog
   and :class:`~sklearn.decomposition.TruncatedSVD`. :pr:`21334` by
   `Thomas Fan`_.
 
+- |API| :func:`decomposition.FastICA` now supports unit variance for whitening.
+  The default value of its :code:`whiten` argument in will change from
```

Comment:
If I am not wrong the single backtick should be the same as the `:code:` directive, isn't it?

```suggestion
  The default value of its :code:`whiten` argument will change from
```